templates/swagger/v1_json.tmpl

```diff
@@ -21,7 +21,7 @@
     },
     "version": "1.1.1"
   },
-  "basePath": "{{AppSubUrl}}/api/v1",
+  "basePath": "/api/v1",
```

Comment:
You need to revert this line I'm afraid

Reply:
Will do. Do you know why that {{AppSubUrl}} got removed from me running the `make generate-swagger` command? Want to make sure it doesn't happen again.